src/main/java/uk/gov/pay/api/model/CardPayment.java

```diff
@@ -60,6 +66,11 @@ public CardDetails getCardDetails() {
         return cardDetails;
     }
 
+    @ApiModelProperty(example = "en")
```

Comment:
Maybe adding `dataType` will make it a little more consistent with the rest and will add a little more info regarding the type